src/System.Private.CoreLib/shared/System/Diagnostics/Tracing/EventSource.Windows.cs

```diff
@@ -0,0 +1,15 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#if ES_BUILD_STANDALONE
+namespace Microsoft.Diagnostics.Tracing
+#else
+namespace System.Diagnostics.Tracing
+#endif
+{
+    public partial class EventSource : IDisposable
+    {
+        internal static uint GetCurrentProcessId() => Interop.Kernel32.GetCurrentProcessId();
```

Comment:
It does not sound right for `EventSource` to be the place one goes to get the current process ID.

I would put the shared stub into `Interop` file, e.g. similar to what we do for RandomBytes:

https://github.com/dotnet/coreclr/blob/master/src/System.Private.CoreLib/shared/Interop/Windows/BCrypt/Interop.BCryptGenRandom.GetRandomBytes.cs#L11
https://github.com/dotnet/coreclr/blob/master/src/System.Private.CoreLib/shared/Interop/Unix/System.Native/Interop.GetRandomBytes.cs#L17